trace-to-events/src/pulse_detection/detectors/threshold_detector.rs

```diff
@@ -128,13 +106,13 @@ impl<Class: ThresholdClass> Assembler for ThresholdAssembler<Class> {
 
 #[cfg(test)]
 mod tests {
-    use super::*;
-    use crate::pulse_detection::EventFilter;
+    use super::{Data, ThresholdDetector, ThresholdDuration};
```

Comment:
```suggestion
    use super::*;
```

I'd prefer to keep this style throughout the codebase.